src/Symfony/Component/DependencyInjection/ConfigurationResolverTrait.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection;
+
+use Symfony\Component\ExpressionLanguage\Expression;
+
+trait ConfigurationResolverTrait
+{
+    protected function resolve($value)
+    {
+        if (is_string($value) && strlen($value) > 0 && $value[0] == '@') {
+            if (strlen($value) > 1 && $value[1] == '=') {
+                return new Expression(substr($value, 2));
+            } else {
```

Comment:
You misread my comment, that `else` is useless due to fact in `if` before you call `return`.